pygeos/test/test_readers_writers.py

```diff
@@ -0,0 +1,52 @@
+import numpy
+import pygeos
+import pytest
+
+from pygeos.ufuncs import from_wkt, from_wkb, to_wkt, to_wkb
+
+
+POINT11_WKB = (
+    b"\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\xf0?\x00\x00\x00\x00\x00\x00\xf0?"
+)
+
+
+def test_from_wkt():
```

Comment:
There is some of overlap with the wkb/wkt tests in `test_geometry`, ideally these should be merged